src/OpenGL/Silk.NET.OpenGL/GL.cs

```diff
@@ -13,26 +18,77 @@ namespace Silk.NET.OpenGL
 {
     public partial class GL
     {
-        [Obsolete
-        (
-            "Parameterless GetApi calls are deprecated and will be removed in a future release. Please create" +
-            "your GL instances using a context"
+        [Obsolete(
+            "Parameterless GetApi calls are deprecated and will be removed in a future release. Please create" + "your GL instances using a context"
         )]
-        public static GL GetApi() => LibraryLoader<GL>.Load
-            (new GLCoreLibraryNameContainer(), SilkManager.Get<GLSymbolLoader>());
-
-        public static GL GetApi(IGLContextSource contextSource) => GetApi
-        (
-            contextSource.GLContext ?? throw new InvalidOperationException
-                ("The given IGLContextSource is not configured with a context.")
-        );
-
-        public static GL GetApi(IGLContext ctx) => GetApi((INativeContext)ctx);
+        public static GL GetApi() => LibraryLoader<GL>.Load(new GLCoreLibraryNameContainer(), SilkManager.Get<GLSymbolLoader>());
+
+        /// <summary>
+        ///     Creates a <see cref="GL" /> instance from an <see cref="IGLContextSource" />.
+        /// </summary>
+        /// <param name="contextSource">
+        /// <see cref="IGLContextSource" /> to create <see cref="GL" /> from.
+        /// </param>
+        /// <returns>
+        ///     A <see cref="GL" /> instance.
+        /// </returns>
+        public static GL GetApi(IGLContextSource contextSource) => GetApi(contextSource.GLContext ?? throw new InvalidOperationException(
+                                                                              "The given IGLContextSource is not configured with a context."));
+
+        /// <summary>
+        ///     Creates a <see cref="GL" /> instance from an <see cref="IGLContext" />.
+        /// </summary>
+        /// <param name="ctx">
+        ///     <see cref="IGLContext" /> to create <see cref="GL" /> from.
+        /// </param>
+        /// <returns>
+        ///     A <see cref="GL" /> instance.
+        /// </returns>
+        public static GL GetApi(IGLContext ctx) => GetApi((INativeContext) ctx);
+
+        /// <summary>
+        ///     Creates a <see cref="GL" /> instance using a function that returns a native OpenGL context.
+        /// </summary>
+        /// <param name="getProcAddress">
+        /// <para>
+        ///     Function returning a native OpenGL context.
+        /// </para>
+        /// <para>
+        ///     The <c>string</c> parameter of the <paramref name="getProcAddress" /> function should be the name of the native function with needs to be called.
```

Comment:
This change goes against the code style for the long line limit of 120 characters. You can solve this using:
```suggestion
        ///     The <c>string</c> parameter of the <paramref name="getProcAddress" /> function should be the name
        ///     of the native function with needs to be called.
```